src/Components/CreatePlayList.js

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from "react";
+import { useData } from "../Context/DataProvider";
+import { checkIfVideoIsAddedToPlaylist } from "../utils";
+export function CreatePlayList() {
+  const [playlistName, setPlaylistName] = useState("");
+  const [createPlaylist, setCreatePlaylist] = useState(false);
+  const {
+    state: { playlist, videoId },
+    dispatch
+  } = useData();
+  function createPlayList() {
```

Comment:
Hey, Please take a look at the playlist case here. i think it should be Playlist()

Reply:
Please reflect this change everywhere applicable.